package.json

```diff
@@ -9,7 +9,7 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "deploy": "firebase deploy"
+    "deploy": "ng build --prod && firebase deploy"
```

Comment:
Can we dogfood ng deploy? This should be enough:

```
ng add @angular/fire
ng deploy
```

Reply:
@mgechev  yes! can you send a PR plz? (the pr should likely remove firebase-tools as I assume that @angular/fire will bring its own version)

I wasn't sure how the whole deploy worked, so I started off by formalizing the process as it originally worked.

Reply:
Here it is https://github.com/StephenFluin/angular-update-guide/pull/47